packages/react-core/src/components/Checkbox/examples/Checkbox.md

```diff
@@ -20,6 +20,20 @@ propComponents: ['Checkbox']
 
 ```
 
+### Reversed
+
+```ts file="./CheckboxReversed.tsx"
+
+```
+
+### Label wraps
+
+When the input is wrapped in a label, larger area can be clicked to check the box, including the space between the checkbox and its description.
```

Comment:
This will be true even for a non-wrapped input, if the `id` prop is passed in. The/one of the main benefits for wrapping the input in a label is that we don't need to pass an `id` into the input nor the `for` attribute to the label. 

```suggestion
When the input is wrapped by a `label` element, an `id` does not need to be passed into the input and the `label` element does not need a `for` prop passed in.
```

cc @edonehoo wdyt of the above?

Reply:
Our recent content research study suggested that people liked having a non-technical description ahead of tech specifics, so I would also consider adding something like _"You can increase the size of a checkbox's clickable area to span wider than the checkbox label."_ and a line break prior to this sentence. 

My usual string of multiple questions: Should we also mention the `isLabelWrapped` prop? Does this prop handle what you described behind the scenes? Where does the `for` attribute come into play (since I'm not seeing it in the other examples to contrast)? 

If so, it may be more helpful to say something like:

_"To expand the clickable area, add the `isLabelWrapped` prop. This wraps the checkbox input with a `label` element, which removes the need to pass `id` to the `<Checkbox>` component"_

(of course, all of this ^ may need to be combined with what you said/remixed for accuracy)

Reply:
@edonehoo the checkbox and radio components are a little confusing at first I think.

The `for` property doesn't need to be passed in, but rather passing in an `id` will set the `for` property internally. So something like `<Checkbox id="test" labelText="Something" />` would render:

```html
<label for="test">Something</label>
<input type="checkbox" id="test" />
```
(or something like that)

I think your last suggestion looks good, maybe just rearranging a little: "To expand the clickable area without needing to pass an `id` in, add the isLabelWrapped prop." 

Not a blocker, though, more nitpicky 😆 I may be overly worried about conveying the idea that "the only way to expand the clickable area is with isLabelWrapped"

Reply:
ah, I see! Thanks for explaining! In that case, I think your rearranged suggestion sounds great

Reply:
@thatblindgeye 

> This will be true even for a non-wrapped input, if the id prop is passed in.

Is the behavior when passing `id` vs `isLabelWrapped` really the same? What I meant by the description is that the **space** between the text and checkbox is also clickable.

Here is an example with using id:

https://github.com/patternfly/patternfly-react/assets/84135613/727db553-a045-4c77-a492-ffcbe6ecac5f

And here with isLabelWrapped:

https://github.com/patternfly/patternfly-react/assets/84135613/f9bbaf9d-80ed-4f78-bf48-6afad08099a4

Good example is also this: https://ux.stackexchange.com/a/47920

I really like both your's and Erin's descriptions, I just thought we should say why is wrapping in a label different than using just the `id` (with the internal `for` attribute)

Reply:
Hm if the space between the checkbox and label is specifically important to call out, then I think that sounds good. I stuck with the more generic "expand the clickable area" because it seems like you can also click to the right of the label to select it, so I wanted to cover both cases. WDYT? 

I would definitely add an example description, so putting it all together, maybe something like this: 

> You can expand the clickable area of a checkbox so that it spans wider than the checkbox label. This allows users to select a checkbox by clicking the checkbox itself, the label, or the area between the checkbox and the label. 
> 
> To expand the clickable area, add the `isLabelWrapped` property.